### PR TITLE
Changed title of globalization topic

### PR DESCRIPTION
**Pull request by rpetrusha**
## Changed title of globalization topic

This PR:
- Changes the title of the top-level globalization and localization topic to apply to .NET and not only .NET Framework. (See #5867)
- Replaces references to .NET Framework with .NET where appropriate.

I haven't made parallel changes to child topics, since these require some revision rather than simple replacement.

/cc @mairaw

### CODE REVIEW

File: docs/standard/globalization-localization/index.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   - "world-ready applications"
-  - "application development [.NET Framework], globalization"
+  - "application development [.NET], globalization"
   - "multilingual application development"
 ms.assetid: 9a59696b-d89b-45bd-946d-c75da4732d02
 author: "rpetrusha"
 ms.author: "ronpet"
 ---
-# Globalizing and Localizing .NET Framework Applications
+# Globalizing and Localizing .NET Applications
```
Comment (mairaw) [inline docs/standard/globalization-localization/index.md]: nit: sentence-case for title and headings

**Review by mairaw — APPROVED** [review]
Good! We make small progress, one topic at a time! 😄 
Left just a small nit comment if you want to address that with this PR.

**Comment by mairaw** [comment]
should we also check incoming links?

**Comment by rpetrusha** [comment]
There are none in the dotnet/docs repo, @mairaw; all references are to "Globalization and Localization." I'm checking dotnet/dotnet-api-docs now.

**Pull request merged**